index.html

```diff
@@ -3,25 +3,26 @@
 description: Victoria BC's programmers whisky society. Connecting technical people through whisky to share their passions and pick up new technologies.
 ---
 
-<h4>Our latest Announcement:</h4>
+<h3 style="text-align: center;"><span style="text-size: +2">{</span> Who Is Whisky Oriented Development? <span style="text-size: +2">}</span></h3>
+<p>
+  We're people who work in the tech industry in Victoria, Canada.  We organize periodic whisky tasting events that seat 30 people.  In each tasting a mix of newcomers and regulars get a chance to sample some fine whisky, learn a little bit about it, and socialize.  We're also a <a href="https://whiskydev.slack.com/messages/general/">Slack</a> channel where people hang out everyday.
+</p>
+
+<h3 style="text-align: center;"><span style="text-size: +2">{</span> Who Can Come? <span style="text-size: +2">}</span></h3>
+<p>
+  Anyone in tech who wants to have nerdy conversations and drink whisky!  If you can sit in a chair and drink liquid then you're adequately prepared.  Whisky is a good social lubricant and it's easy to make friends in the relaxed atmosphere.
+</p>
+
+<h3 style="text-align: center;"><span style="text-size: +2">{</span> How Can I Get In Touch? <span style="text-size: +2">}</span></h3>
+<p>
+  For distributor and vendor communications talk to <a href="mailto:graham@whiskydev.com">Graham</a>.  For social media inquiries talk to <a href="mailto:jamie@whiskydev.com">Jamie</a>.  For questions about past or future tastings talk to <a href="mailto:jeff@whiskydev.com">Jeff</a>.
```

Comment:
Yeah, I was talking with Jeff today about maybe setting up custom email forwarding addresses, and just having those instead.